charts/generic-data-analytics-extractor/templates/_helpers.tpl

```diff
@@ -17,3 +17,17 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
```

Comment:
Is this block needed, it looks like the logic here could be worked into the section above?

Reply:
I'm not sure I follow. Aren't they 2 separate things?
Isn't the first block (app.fullname) some kind of DNS name, truncated at 63 chars?

And the new block I've added is truncated at 52 chars and is specifically for the resultant resource name in k8s ?

Reply:
So the 52 chars limit only applies to cronjob names,  here's the explanation: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#:~:text=Even%20when%20the%20name%20is,no%20more%20than%2063%20characters.

Suggest you rename the named template to reflect this and update the comment also. e.g. `generic-data-analytics-extractor.cronjob-name`

Reply:
Ah ha, that makes sense. Perfect, will do ... 👍

Reply:
@mattops - I think I've addressed this now in my last commit, if you like to take another look please?